flask_appbuilder/security/manager.py

```diff
@@ -701,6 +713,48 @@ def _decode_and_validate_azure_jwt(self, id_token: str) -> Dict[str, str]:
 
         return jwt.decode(id_token, options={"verify_signature": False})
 
+    def _get_authentik_jwks(self, jwks_url) -> dict:
+        import requests
```

Comment:
requests is not an included package, can you change this to use something else like urllib or http.client